features/step_definitions/login_steps.rb

```diff
@@ -0,0 +1,18 @@
+require 'byebug'
+
+When('I fill in Correo with {string} and Contrasenia with {string} and click on {string} button') do |email, password, button|
+  enter_text(:id, email, 'email')
+  enter_text(:id, password, 'password')
+  click(:id, 'unauth_container')
+  click(:id, button)
+  wait(5)
```

Comment:
This is called an implicit wait. We try to avoid this by setting expected conditions (i.e., waiting for an element to be ready). I strongly recommend you to use the methods `wait_for_element_to_display` and `wait_for_element_to_enable`.
You can read more about this [here](https://www.guru99.com/implicit-explicit-waits-selenium.html)